routes/sites.js

```diff
@@ -12,10 +12,8 @@ const {
 const router = Router();
 
 const getSites = (allSites) =>
-  `SELECT site.*, to_json(contact1) as "primaryContactInfo", to_json(contact2) as "secondContactInfo"
-  FROM site
-    INNER JOIN general_user as contact1 ON contact1.user_id = site.primary_contact_id
-    LEFT JOIN general_user as contact2 ON contact2.user_id = site.second_contact_id
+  // `SELECT site.*, to_json(contact1) as "primaryContactInfo", to_json(contact2) as "secondContactInfo"
+  `SELECT site.* FROM site
```

Comment:
can you get rid of this since josh did it a different way that gets the contact info in a nested way (might just be able to merge dev into your branch to fix it)